### PR TITLE
Feat:Contribution section

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [x] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [x] Make MR mark as ready before submitting it for review
- [x] Add images, videos, or gifs of UI 
- [x] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**
- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook  (chrome, firefox, safari)
- [ ] iPhone(safari)
- [x] Available android devices  (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)
**Description :**
Create Contribution section for mobile and desktop.
**Video:**
[screen-capture.webm](https://github.com/canopas/canopas-website/assets/98794636/f29b95a4-3778-4adb-acda-6d9a2c8e9be6)

**Performance Mobile:**
image
**Performance Desktop**
image
Desktop:
image

**Mobile:**
image

### CODE REVIEW

File: vue-frontend/src/components/jobs-thankyou/ContributionSection.vue [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+  data() {
+    return {
+      swiper: null,
+      contributionURL: Config.GITHUB_URL,
+      icon: faGithub,
+      rightArrow: faArrowRightLong,
+      websiteOpenSourceUrl: Config.WEBSITE_OPEN_SOURCE_URL,
+      contributions: [
+        {
+          title: "Intro Showcase view in jetpack compose",
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs-thankyou/ContributionSection.vue:175]: Let's rename it to `Intro Showcase view` only

File: vue-frontend/src/components/jobs-thankyou/ContributionSection.vue [hunk 2]
```diff
@@ -0,0 +1,243 @@
+<template>
+  <div class="tw-my-20 lg:tw-mb-40">
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs-thankyou/ContributionSection.vue:2]: Review the `our success stories` section on the services page on larger(4k) devices, it should like that only

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/jobs-thankyou/ContributionSection.vue [hunk 3]
```diff
@@ -0,0 +1,243 @@
+<template>
+  <div class="lg:tw-container tw-my-20 lg:tw-mb-40">
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs-thankyou/ContributionSection.vue:2]: Not looking good in 992 to 1400 devices, also container should be in > xl or 2xl devices only. Review other sections properly for all devices and implement like those only

**Review by cp-sumi-k — APPROVED** [review]

File: vue-frontend/src/components/jobs-thankyou/ContributionSection.vue [hunk 4]
```diff
@@ -0,0 +1,243 @@
+<template>
+  <div class="xll:tw-container tw-my-20 lg:tw-mb-40">
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs-thankyou/ContributionSection.vue:2]: Still not looking properly. Title should align with other divs as shown in image

Untitled design (1)

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/jobs-thankyou/ContributionSection.vue (hunk 4 above)
Comment (cp-priyanka-g) [reply vue-frontend/src/components/jobs-thankyou/ContributionSection.vue:2]: In figma it is like this only... little bit outer side from perks and benefit section , not aligned in same line .

Comment (cp-sumi-k) [reply vue-frontend/src/components/jobs-thankyou/ContributionSection.vue:2]: But its not same for all width. Can you check for all widths?

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**